docs/release-notes/2022-Q4.md

```diff
@@ -21,6 +21,7 @@ In addition to complete solution for raster product’s consumption. A few major
 #### Catalog Service
 ##### Catalog Profile Update
 Additions:
+  * `WMTS_KVP` - was added to links in order to serve kvp capabilities template 
```

Comment:
this should be in 2023-Q1 release notes, don't edit previous release notes

Reply:
removed from the previous release notes